scripts/release.js

```diff
@@ -111,6 +110,29 @@ fs.writeFileSync(
 spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
 logger.log(`Updated package.json version to ${versionString}`);
 
+
+// Udpate meeting demo to the most up to date version of the SDK
+process.chdir(path.join(__dirname, '../demo/meeting'));
```

Comment:
Need to update chat demo too so this should be a loop. We may also want to update the MCM to verify the chat functionalities.

Reply:
Okay, I'll update the sdk version in the chat demo also